### PR TITLE
Expo font and assets.

### PR DESCRIPTION
**Pull request by janakg**

### CODE REVIEW

**Comment by janakg** [comment]
Hi, Can you please let me know if expo-font needs any additional configuration to make to it work? right now I am getting an error while loading a font. 

Error: 'file:///Volumes/macDrive/Users/janakg/Library/Developer/CoreSimulator/Devic…/ExponentExperienceData/ExponentAsset-cbd50882df82be5fce046e52b302e316.ttf' isn't readable.

**Comment by janakg** [comment]
sjchmiela helped me. I have pushed a working expo-font example. This can be a sample repo to showcase unimodules.  Will add more examples. Thanks.

File: android/settings.gradle [hunk 1]
```diff
@@ -1,7 +1,57 @@
-apply from: '../node_modules/react-native-universal-modules/settings.gradle'
 rootProject.name = 'MyApp'
 
 include ':app'
 
+include ':expo-core'
```
Comment (brentvatne) [inline android/settings.gradle:5]: this isn't needed - the stuff from react-native-universal-modules expands to this. you only need to add expo-font

File: ios/Podfile [hunk 2]
```diff
@@ -18,8 +17,23 @@ target 'MyApp' do
   pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
-  use_universal_modules!
+  pod 'EXCore', path: '../node_modules/expo-core/ios'
```
Comment (brentvatne) [inline ios/Podfile:20]: same comment as above in settings.gradle, you only need to add EXFont here and leave the `use_universal_modules!` bit in place

File: android/settings.gradle (hunk 1 above)
Comment (janakg) [reply android/settings.gradle:5]: Sure. we had file-system issue, so we removed react-native-universal-modules to update the dependencies easily. It was a temp solution until uni modules become stable. Now I will try again with use_unimodules.

**Comment by brentvatne** [comment]
did this actually work for you on android or just ios?

**Comment by janakg** [comment]
> did this actually work for you on android or just ios?

Yes. It works in both.

**Comment by janakg** [comment]
Hi, Merged to your new changes. Another sample repo with font imports.
https://github.com/minewhat/my-app-with-universal-modules

**Comment by brentvatne** [comment]
i updated expo-asset so this now works as expected out of the box. check out the master branch

**Pull request closed without merging**